src/components/Alert/styles.js

```diff
@@ -3,10 +3,10 @@ import IconButton from '../IconButton';
 import { theme } from '../../theme';
 
 const styles = {
-  color({ type }) {
-    if (type === 'error') return theme.color.element.error;
-    if (type === 'success') return theme.color.element.success;
-    if (type === 'information') return theme.color.brand.secondary;
+  color({ alertType }) {
```

Comment:
Why did you rename this?

Reply:
`type` is a valid html value so the dom was getting poluted

Reply:
What about `intent` then? We already use this naming with the Button component

Reply:
this is used only internally within `Alert`, the prop we use stays with the same name, I didn't want to change the name so much, just avoid it from going to the dom